erpnext/public/js/hub/pages/Item.vue

```diff
@@ -203,33 +199,39 @@ export default {
 		},
 
 		add_to_saved_items() {
-			hub.call('add_item_to_user_saved_items', {
-				hub_item_name: this.hub_item_name,
-				hub_user: frappe.session.user
-			})
-			.then(() => {
-				const saved_items_link = `<b><a href="#marketplace/saved-items">${__('Saved')}</a></b>`
-				frappe.show_alert(saved_items_link);
-				erpnext.hub.trigger('action:item_save');
-			})
-			.catch(e => {
-				console.error(e);
-			});
+			hub
+				.call('add_item_to_user_saved_items', {
+					hub_item_name: this.hub_item_name,
+					hub_user: frappe.session.user
+				})
+				.then(() => {
+					const saved_items_link = `<b><a href="#marketplace/saved-items">${__(
```

Comment:
Translated strings on multiple lines won't work. They must be on a single line.

https://frappe.io/docs/user/en/translations#3-blocks